Sources/NIOPosix/MultiThreadedEventLoopGroup.swift

```diff
@@ -299,7 +320,21 @@ public final class MultiThreadedEventLoopGroup: EventLoopGroup {
             }
         }
     }
-
+    
+    #if swift(>=5.7)
+    /// Convert the calling thread into an `EventLoop`.
+    ///
+    /// This function will not return until the `EventLoop` has stopped. You can initiate stopping the `EventLoop` by
+    /// calling `eventLoop.shutdownGracefully` which will eventually make this function return.
+    ///
+    /// - parameters:
+    ///     - callback: Called _on_ the `EventLoop` that the calling thread was converted to, providing you the
+    ///                 `EventLoop` reference. Just like usually on the `EventLoop`, do not block in `callback`.
+    @preconcurrency
+    public static func withCurrentThreadAsEventLoop(_ callback: @escaping @Sendable (EventLoop) -> Void) {
```

Comment:
This does not need to be `@Sendable`: the closure is deterministically invoked in the current thread context.

Reply:
Done in https://github.com/apple/swift-nio/pull/2211/commits/29e15c4d3b0ebb26b61babdde6617f86899462c9